packages/angular-cli/utilities/find-parent-module.ts

```diff
@@ -11,10 +11,9 @@ export default function findParentModule(project: any, currentDir: string): stri
   let pathToCheck = path.join(sourceRoot, currentDir);
 
   while (pathToCheck.length >= sourceRoot.length) {
-    // let files: string[] = fs.readdirSync(pathToCheck);
-
-    // files = files.filter(file => file.indexOf('.module.ts') > 0);
+    // TODO: refactor to not be based upon file name
     const files = fs.readdirSync(pathToCheck)
+      .filter(fileName => !fileName.endsWith('routing.module.ts'))
```

Comment:
There's a scenario where someone has a single module called `routing` and we'll never find it, but tbh I don't think we should special-case it this far... when we refactor we'll handle it.